README.md

```diff
@@ -115,4 +96,4 @@ Moldy is on the following social networks
 
 ---
 
-Made with ❤️ in the world
+<div align="center">Made with ❤️ in the world</div>
```

Comment:
Here make not center the text only in the left side 😄

Reply:
And add the license badge

Reply:
with this should be to lick it